test-node.bash

```diff
@@ -136,6 +141,7 @@ while [[ $# -gt 0 ]]; do
             echo --init:            remove all data, rebuild, deploy new rollup
             echo --pos:             l1 is a proof-of-stake chain \(using prysm for consensus\)
             echo --validate:        heavy computation, validating all blocks in WASM
+            echo --fee-token:       chain is set up to use custom fee token
```

Comment:
we should probably specify this is for L3 only, and only respected when --l3node is set

Reply:
done